classes/04class/exercises/c04-iac02/raghunadhpokkalath/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "availability_zones" {
+  type = list
+}
+
+variable "vpc" {
+  type = map
+}
+
+variable "public_subnet_a" {
+  type = map
+}
+
+variable "public_subnet_b" {
+  type = map
+}
+
+variable "private_subnet_a" {
+  type = map
+}
+
+variable "private_subnet_b" {
+  type = map
+}
```

Comment:
good use of the `map` data type.

For having more control over the input, you can use `map` and `object`:
```
map(object({
    name = string
    cidr = string
})
)
```